### ISSUE TITLE
BPNS has floats in long_store instead of bool in the soft_sediment sheet

### ISSUE THREAD

**Issue by cymon** [emo-bon/observatory-profile#32]
Four floats lines 179-182 in long_store field of EMO_BON_Metadata_Soft_Sediment_BPNS_Belgium_new - should be booleans.

Processing observatory_id='BPNS' - sampling_strategy='soft_sediment' - sheet_type='sampling'
Sample sheet link: https://docs.google.com/spreadsheets/d/1zc0bZdpl-Eoi35lI_5BGkElbscplyQRyNPLkSgeEyEQ/gviz/tq?tqx=out:csv&sheet=sampling
Discarded 11 records leaving 182.
ValidationError: 1 validation error for Model
long_store
  Value error, Unrecognised value: 479.3 [type=value_error, input_value='479.3', input_type=str]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error

**Comment by kmexter** [comment]
yes, they appear to have put the wrong values in these cells (179-182 AB) - @melinalou can you ask them to correct that?

**Comment by melinalou** [comment]
yes I will find the contact and ask them to do it.

**Comment by isanti** [comment]
this is fixed
